core/app/models/spree/credit_card.rb

```diff
@@ -8,8 +8,8 @@ class CreditCard < Spree::Base
 
     after_save :ensure_one_default
 
+    attr_reader :number
     attr_accessor :encrypted_data,
-                    :number,
```

Comment:
how is the number supposed to be written to the credit card object without a write accessor?